idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -236,39 +236,37 @@ def run_star_part(output_dir, genome_dir, fastq_files):
     execute_command_realtime_stdout(" ".join(star_command_params), os.path.join(output_dir, "Log.progress.out"))
 
 def run_star(fastq_files):
-    # check if genome downloaded already
-    genome_file = os.path.basename(STAR_GENOME)
-    if not os.path.isfile("%s/%s" % (REF_DIR, genome_file)):
-        execute_command("aws s3 cp %s %s/" % (STAR_GENOME, REF_DIR))
-        execute_command("cd %s; tar xvfz %s" % (REF_DIR, genome_file))
+    star_outputs = [STAR_OUT1, STAR_OUT2]
+    num_fastqs = len(fastq_files)
+    def unmapped_files_in(some_dir):
+        return ["%s/Unmapped.out.mate%d" % (some_dir, i+1) for i in range(num_fastqs)]
```

Comment:
Please verify that when the input is only 1 fastq file, the output is still Unmapped.out.mate{1/2} 
It's possible it's just Unmapped.out

Reply:
Interesting.   I just preserved the original semantics here, that was what the code before my change was assuming.

Reply:
I checked it and it is unmapped.out.mate1 for single-end case